### PR TITLE
feat!: generate with microgenerator

### PR DESCRIPTION
**Pull request by busunkim96**
~~*Blocked*~~ https://github.com/googleapis/proto-plus-python/issues/81 is resolved. Stringy enums are now convertible, allowing the library to turn proto in JSON back into library types. 🥳

### CODE REVIEW

File: README.rst
Comment (arithmetic1728) [inline README.rst:55]: supported python version above = 3.6?

File: setup.py
Comment (arithmetic1728) [inline setup.py:65]:
```suggestion
        "Programming Language :: Python :: 3.7",
        "Programming Language :: Python :: 3.8",
```

**Review by arithmetic1728 — APPROVED** [review]
Seems the samples are failing, not sure why.

**Comment by busunkim96** [comment]
All commits were made by Googlers with valid CLAs.

**Review by arithmetic1728 — APPROVED** [review]

**Comment by busunkim96** [comment]
@leahecole When you have a moment please review the samples https://github.com/googleapis/python-securitycenter/pull/49/files#diff-929314196ba76c84956d324d5c95111e

Thank you!

**Review by software-dov — APPROVED** [review]
LGTM

**Review by leahecole — COMMENTED** [review]
samples LGTM to me if tests are passing with the caveat that I am super unfamiliar with this product

**Comment by busunkim96** [comment]
@leahecole Thanks for the review! Could you "approve" via the GItHub UI? (it looks like I am blocked from merging b/c of the Codeowners requirement). 

Tests are passing: https://source.cloud.google.com/results/invocations/f262a36c-c597-4aed-bf9b-07c5c32aa3f3/targets/github%2Fpython-securitycenter%2Fsamples%2Fsnippets/tests 😁

**Review by leahecole — APPROVED** [review]

**Comment by leahecole** [comment]
Yup sorry about that!

**Pull request merged**
